src/ray/protobuf/gcs_service.proto

```diff
@@ -667,6 +667,8 @@ message GetTaskEventsRequest {
     optional string name = 4;
     // True if task events from driver (only profiling events) should be excluded.
     optional bool exclude_driver = 5;
+    // True if filtering by tasks paused by the debugger
+    optional bool is_debugger_paused = 6;
```

Comment:
is this needed since we are not pushing down filters to GCS?

Reply:
yes, I use it here: https://github.com/ray-project/ray/pull/41067/files#diff-bb7747741306d767d5a35c0e55cde709d54cc407b24c806a61d63392795e1303R350

Reply:
like on the caller side, we don't have code that sets it anymore? I beleive it was needed in earlier iterations here: https://github.com/ray-project/ray/pull/41067#discussion_r1388743316

Reply:
got it, removed.